services/metric_service.go

```diff
@@ -90,7 +90,7 @@ func NewMetricService(l log.Logger, addr string, token string, enabled bool, reg
 }
 
 // StoreMetrics aggregates and stores metrics
-func (m MetricService) StoreMetrics(req domain.MetricsRequest) error {
+func (m MetricService) StoreMetrics(_ context.Context, req domain.MetricsRequest) error {
```

Comment:
how come we added a context thats never used?

Reply:
never mind, i realise now

Reply:
It's used by the `MetricsStream` which also has the `StoreMetrics` method so I had to add one here to make it implement the same interface but then the linter was flagging it as not being used. I'm hoping there's some refactoring I can do in here where we might be able to use this context rather than just having it to satisfy the interface.